<?xml version="1.0"?>
<launch>

	<!-- joy node -->
	<node pkg="joy" type="joy_node" name="ps3_joy" output="screen" respawn="false">
		<param name="dev" type="string" value="/dev/input/js0" />
		<param name="deadzone" value="0.12" />
		<param name="autorepeat_rate" value="10.0" type="double" />
	</node>

	<!-- Parameters -->
	<param name="cmd_topic_vel" value="/summit_xl_controller/command" type="string"/>
	<param name="cmd_topic_ptz" value="/axis/cmd_ptz" type="string"/>
	<rosparam command="load" file="$(find summit_xl_teleop)/config/ps3.yaml" />
	<node name="summit_xl_teleop" pkg="summit_xl_teleop" type="summit_xl_teleop" output="screen"/>

	<!-- Rviz -->
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find summit_xl_teleop)/rviz/summit.rviz"/>

</launch>
